MSAL/src/public/MSALError.h

```diff
@@ -100,6 +100,12 @@ typedef NS_ENUM(NSInteger, MSALErrorCode)
     MSALErrorAuthorizationFailed = -42104,
     
     /*!
+        MSAL received a valid token response, but it didn't contain an access token.
+        By Oauth 2.0 spec it should never happen, but it still occasionally happens.
```

Comment:
Replace "but it still occasionally happens" with, "check to make sure your application is consented get all of the scopes you are asking for."